dashboard/src/Entity/Event.php

```diff
@@ -8,15 +8,10 @@
 
 /**
  * @ORM\Entity
- * @ORM\Table
+ * @ORM\Table(uniqueConstraints={@ORM\UniqueConstraint(columns={"run", "type"})})
```

Comment:
de-duplication should be dealt with by the `@Id` field

Reply:
It'll probably become a constraint on `run`, `type`, and `attempt`. But let's ignore for now (since this is already simpler than we know it needs to be).